### DISCUSSION TITLE
About batch size

### DISCUSSION THREAD

**Discussion by cdefghijkl** [TheLastBen/fast-stable-diffusion#1218] [General]
Do concept image steps batch size still at 1 eventhough I change the code to 8?

**Comment by TheLastBen** [comment]
the settings for concept images are inside the function `dump_only_textenc()`

**Reply by cdefghijkl** [reply]
How many concept images do you recommend for 160 instance images?

**Reply by TheLastBen** [reply]
it depends how much overfitting you're getting, if the results are diverse, you don't need to regularize with concept images at all
